frontend/src/store/modules/app.js

```diff
@@ -67,6 +68,12 @@ export default {
         state.districtCodes.add(element.mincode?.substring(0, 3));
       });
     },
+    setActiveSchools(state, activeSchools) {
+      state.activeSchoolMap = new Map();
+      activeSchools.forEach(element => {
+        state.activeSchoolMap.set(element.schoolID, element);
+      });
```

Comment:
hmm if we are going to have a map anyhow, should we do this logic in the NodeJS? following the pattern for schoolMap. 

Then the frontend users don't have to loop through all the schools each time they load up the webpage.